03_probability_intro/03_probability_intro.jl

```diff
@@ -397,6 +397,14 @@ md"
 Considering that we have only tried 100 times, the probabilities have been estimated pretty well! Each distribution assigns a high-enough probability to the true value of the bandit probabilities and it's surroundings. Another strategies rather than the Thompson sampling can be tested to see how well they perform, this was just a simple example to apply Bayesian probability.
 "
 
+# ╔═╡ 49b84f96-7c2f-11eb-3bf8-bde596991317
+md"
+### Summary
+In this chapter we start showing the basics concepts of statistics.
+Then we see how we can approach problems in a Bayesian way. How our first beliefs on a hypothesis can be updated with the income of new data and the most common probability distribution plots, that is the way bayesians represent probability.
+At last, we solve the multi-armed bandit problem thinking in a Bayesian way.
```

Comment:
I think it will be better to change 
> At last,

to
> Finally

or something similar.

Reply:
check tense

Reply:
try giving some little more detail about the problem context and solution in a very high-level.